schema/crdb/bgp-oxpop-features/up04.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE omicron.public.switch_port_settings_bgp_peer_config ADD COLUMN IF NOT EXISTS md5_auth_key TEXT;
```

Comment:
`up03` and `up04` look like they're the same?

Reply:
Yeah, removed one.